content/W2017/W17-tshirt-contest.md

```diff
@@ -0,0 +1,33 @@
+Title: T-Shirt Contest
+Date: 2017-02-26
+Category: Events
+Tags: social, big csters, panel
+Slug: tshirt-contest-w17
+Author: Monica Xu
+Summary: Design a T-Shirt for WiCS 
+
+
+Have a passion for design? 
+Love fashion? 
+What to have your designs worn by everyone around campus? 
+
+Submit a design to
+the WiCS T-Shirt Contest... you could be the next trendsetter!
+
+Designs must adhere to the following:
++  To be inclusive of women and non-binary individuals, designs
+   cannot include gender-specific symbols
++  Fit on A4 paper
++  1 or 2 colours on 1 solid colour shirt of choice
++  Include "Women in Computer Science" or "WiCS"
++  Be appropriate and adhere to the UW and WiCS codes of conduct
+
+Deadline is February 26! Submit to wics-ugrad@lists.uwaterloo.ca
+
+
+## Contest Details ##
+
++ **Who:** Women and Non-Binary students in the Faculty of Math
```

Comment:
oops my bad

okay the problem is that you need a new line before the list starts